code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -1251,7 +1251,7 @@ Defined in conflicts.dm of the #defines folder.
 		G.aim_slowdown -= SLOWDOWN_ADS_SCOPE
 		G.wield_delay -= WIELD_DELAY_FAST
 		bipod_deployed = !bipod_deployed
-	else if(do_after(user, 10, TRUE, 5, BUSY_ICON_BUILD))
```

Comment:
Is the removal of the icon intended here?

Reply:
Yea.